mainnet-contracts/script/DeployPufETHBridging.s.sol

```diff
@@ -41,52 +40,54 @@ contract DeployPufETHBridging is BaseScript {
         //@todo this is for tests only
         AccessManager(deployment.accessManager).grantRole(1, _broadcaster, 0);
 
-        xPufETH xpufETHImplementation = new xPufETH();
+        // xPufETH xpufETHImplementation = new xPufETH();
 
-        xPufETH xPufETHProxy = xPufETH(
-            address(
-                new ERC1967Proxy{ salt: bytes32("xPufETH") }(
-                    address(xpufETHImplementation), abi.encodeCall(xPufETH.initialize, (deployment.accessManager))
-                )
-            )
-        );
+        // xPufETH xPufETHProxy = xPufETH(
```

Comment:
You can just remove the xPufETH references completely.

This script is unlikely to be used, but its nice to have to deploy to a testnet (with LZ)